bootloader/rust-toolchain

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "nightly-2021-03-25"
```

Comment:
if this isn't the same toolchain that we use for the firmware, be careful that it requires a "hack" in the github workflow to force its installation.

Reply:
I'd leave this to my future self to decide if I want to also upgrade the bootloader toolchain or change the workflows instead. The breaking workflows will be pretty hard to miss, so I think this can be left as is?